prog-template.c

```diff
@@ -16,6 +16,7 @@
 #define CLIENT_PORT 2000
 #define MAXLINE 1024 
 #define KH4_GYRO_DEG_S   (66.0/1000.0)
+#define LRF_DEVICE "/dev/ttyACM0" 
```

Comment:
will the LRF_DEVICE path always be the same on every robot? If not how can we automate setting this path?